contents/founders/how-come-we-ship-so-much.md

```diff
@@ -0,0 +1,136 @@
+---
+date: 2023-10-30
+title: "We like to ship fast. Here's how we designed our company to do it"
+author:
+  - james-hawkins
+featuredImage: ../images/blog/super-hog-pink.png
+featuredImageType: full
+tags:
+  - Founders
+  - Product engineers
+  - Engineering
+  - Culture
+crosspost:
+  - Product engineers
+---
+
+Clearly there are exceptions, but we ship a lot faster than the average company.
+
+Our first line of code was January 22nd, 2020, after a pivot during a YC batch. Since then, we've shipped:
+
+* [Product analytics](/product-analytics)
+* [Session replay](/session-replay)
+* [Feature flags](/feature-flags)
+* [A/B testing](/ab-testing)
+* [User surveys](/docs/surveys)
+* Heatmaps
+* [SDKs](/docs/libraries) for all the above
+* A basic [CDP](/docs/cdp) to stream data to warehouses
+* [Data warehouse](/docs/data-warehouse) MVP
+* A wildly extensive website
+
+...and those are just the things that worked.
+
+All the above are being used a lot, too. We're ingesting 10s of billions of events a month, have 10s of thousands of companies regularly using our service, and over 100k user signups.
+
+I often get asked how we build so much, so quickly. The answer is simple: we deliberately designed our company for speed.
```

Comment:
```suggestion
I often get asked how we build so much, so quickly. The answer is simple: we deliberately designed our company for speed.  We did this because we want to build [lots of products in one](/handbook/why-does-posthog-exist) - it's a lot to ship!
```